shell/assets/translations/en-us.yaml

```diff
@@ -2568,6 +2568,16 @@ fleet:
     tabs:
       restrictions: Allowed Target Namespaces
     timeout: Workspace creation timeout. It's possible the workspace was created. We suggest checking the workspace page before trying to create another.
+    remove:
```

Comment:
I'm not quite sure this is clear enough (and one part is wrong).

Priority # 1 - This will delete cluster objects. If rancher manages the cluster lifecycle it will also delete the actual cluster (not just rancher CRs). For example a GKE cluster would be deleted and cannot be re-imported. I think we need to consult our local innovation champion / @manno ... this is correct right? if i were to delete the `fleet-default` namespace and it contained a v1 prov cluster it would be the same as just deleting the v1 prov cluster which would delete the underlying cluster and nodes?

Priority # 2 - The workspace contains a set of fleet resources, like gitrepos, which will all be deleted when the workspace is deleted.

Reply:
@richard-cox I changed the message:

![image](https://github.com/user-attachments/assets/a8feab2f-362f-4d24-a55e-f6b554af300b)

We can change the label to include more information, but I don't know what exactly.
cc @manno

Reply:
![image](https://github.com/user-attachments/assets/41a805a8-bb19-4c5b-830a-edbcaf6b1770)

Reply:
@kinarashah Would you be able to advise? What happens to provisioned artefacts (vm's, hosted cluster, etc) if a the namespace containing the v1 provisioning cluster is deleted? Would they be removed or left orphaned?

Reply:
Kinara believes so, and Brad has confirmed. Deleting the namespace a v1 prov cluster is in is the same as deleting each cluster individually (infrastructure is removed).

```
Deleting a Workspace also deletes all resources in the Workspace. This includes GitRepos but also Clusters.

Deleted Clusters will be removed from Rancher, and if Rancher manages the life-cycle of the Cluster it's underlying infrastructure will also be taken down.
```

i think we replace Rancher with a string substitute for white labelling, so it wouldn't be the exact `Rancher` word in the above quote